src/Http/Middleware/BaseMiddleware.php

```diff
@@ -24,16 +25,12 @@ abstract class BaseMiddleware
 {
     /**
      * The JWT Authenticator.
-     *
-     * @var JWTAuth
      */
-    protected $auth;
+    protected JWTAuth $auth;
```

Comment:
Shouldn't be a problem, or? What if someone overrode this Middleware and also defined this property? 🤔

---
Note: _Guess it's fine_, also this isn't the only property being typed now in this PR but probably also something for the changelog.